docs/advanced-usage.rst

```diff
@@ -448,3 +448,34 @@ To enable this simply define environment variable `SSLKEYLOGFILE`:
 
 Then configure the key logfile in `Wireshark <https://wireshark.org>`_, see
 `Wireshark TLS Decryption <https://wiki.wireshark.org/TLS#TLS_Decryption>`_ for instructions.
+
+Custom SSL Contexts
+-------------------
+
+You can exercise fine-grained control over the urllib3 SSL configuration by
+providing a :class:`ssl.SSLContext <python:ssl.SSLContext>` object. For purposes
+of compatibility, we recommend you obtain one from
+:func:`~urllib3.util.ssl_.create_urllib3_context`.
+
+Once you have a context object, you can mutate it to achieve whatever effect
+you'd like. For example, the code below loads the default SSL certificates, sets
+the :data:`ssl.OP_NO_TICKET<python:ssl.OP_NO_TICKET>` flag that isn't set by
```

Comment:
[The default `create_ssl_context` does set `OP_NO_TICKET`](https://github.com/urllib3/urllib3/blob/main/src/urllib3/util/ssl_.py#L258), maybe we should change this example?

Reply:
Ugh, right, that's why we didn't include it in the first place. I'm actually surprised to see all the things that can be done without a custom SSL Context.

Honestly, the reason most users wanted to do that in requests was to add back an insecure cipher, but I'm not sure you'll be happy to see that documented. :D

The other thing left are verify_flags, which is what prompted this whole thing, but I don't understand well what they do.

Reply:
If we're just looking for an option we can use as an example, maybe `OP_CIPHER_SERVER_PREFERENCE`?

Reply:
Thanks, fixed! It's only available starting with Python 3.8+, but at least it's not set by default.